modules/rest/src/main/java/com/enonic/xp/app/contentstudio/rest/resource/content/json/ContentSelectorQueryJson.java

```diff
@@ -36,7 +39,8 @@ public ContentSelectorQueryJson( @JsonProperty("queryExpr") final String queryEx
                                      @JsonProperty("inputName") final String inputName,
                                      @JsonProperty("contentTypeNames") final List<String> contentTypeNamesString,
                                      @JsonProperty("allowedContentPaths") final List<String> allowedContentPaths,
-                                     @JsonProperty("relationshipType") final String relationshipType )
```

Comment:
there is a chance that `relationshipType` is not used any more

Reply:
this is not a part of this task